### ISSUE TITLE
ProgressIndicator: current position not clearly visible in high contrast.

### ISSUE THREAD

**Issue by aneeshack4** [OfficeDev/office-ui-fabric-react#10849]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Bug1 of #8388: Current position in progress Indicator is not that much prominent and clear in High contrast modes.

### Please provide a reproduction of the issue in a codepen:
![image](https://user-images.githubusercontent.com/4161791/66875819-9e761500-ef7d-11e9-8cba-7c4eb2ce5c86.png)

#### Actual behavior:

The current progress flashes by in white on a white bar so the lack of contrast makes it not very visible.

#### Expected behavior:

Should be blue or some other color in HC mode to make it easy to distinguish.
